### PR TITLE
Nemo/Out of service mode toggled per individual application

### PR DESCRIPTION
**Pull request by nemodjuric**
Connected to #2702 

Controllers responsible for application requests are:

- Certification:

 certifications_controller.rb, 

- Reader:

 reader/application_controller.rb, 

- Hearings prep:

 hearings_controller.rb, 

- Dispatch:

 tasks_controller.rb

out

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,12 +45,12 @@ First time user will need to run `create` before `setup`:
 
 ```
 docker-compose run rake db:create
-```
```
Comment (nemodjuric) [inline README.md]: I suspect my Sublime got rid of these automatically.

Comment (amprokop) [inline README.md]: ok looks fine

File: README.md [hunk 2]
```diff
@@ -273,6 +273,28 @@ FeatureToggle.details_for(:apple)
...
+# enable for dispatch only
+Rails.cache.write("dispatch_out_of_service", true)
+
+# enable for hearings only
+Rails.cache.write("hearings_prep_out_of_service", true)
+
+# enable for reader only
+Rails.cache.write("reader_out_of_service", true)
+
+# to disable, i.e.
```
Comment (amprokop) [inline README.md]: this is a very very small nit but you'd want `e.g.` (which means "for example"), or maybe just delete `i.e.`, you don't need it. it took me like 3 years of trying to remember these latin abbreviations to get it right lol.

File: app/controllers/reader/application_controller.rb [hunk 3]
```diff
@@ -20,4 +20,10 @@ def verify_reader_feature_enabled
   def verify_access
     verify_authorized_roles("Reader")
   end
+
+  private
+
+  def check_reader_out_of_service
```
Comment (amprokop) [inline app/controllers/reader/application_controller.rb]: fyi @mdbenjam, let us know if you foresee any problems with this.

File: app/controllers/tasks_controller.rb [hunk 4]
```diff
@@ -3,9 +3,14 @@
 #       from EstablishClaimsController can be abstracted out
 class TasksController < ApplicationController
   class TaskTypeMissingError < StandardError; end
+  before_action :check_dispatch_out_of_service
 
   private
 
+  def check_dispatch_out_of_service
```
Comment (amprokop) [inline app/controllers/tasks_controller.rb]: fyi @sharonwarner, let me know if you see any problems with this

File: app/controllers/hearings_controller.rb [hunk 5]
```diff
@@ -17,6 +17,10 @@ def logo_path
 
   private
 
+  def check_hearings_prep_out_of_service
```
Comment (amprokop) [inline app/controllers/hearings_controller.rb]: fyi @sharonwarner, let us know if you see any problems with this

File: app/controllers/reader/application_controller.rb (hunk 3 above)
Comment (mdbenjam) [inline app/controllers/reader/application_controller.rb]: I don't see any problems with this 👍

File: app/controllers/hearings_controller.rb (hunk 5 above)
Comment (sharonwarner) [inline app/controllers/hearings_controller.rb]: 👍

File: app/controllers/tasks_controller.rb (hunk 4 above)
Comment (sharonwarner) [inline app/controllers/tasks_controller.rb]: 👍

**Pull request merged**